File: ABC-227/C.cpp
#include <iostream>

using namespace std;

int main() {
  long long n;
  cin >> n;
  long long ans = 0;
  for (long long i = 0; i <= n; i++) {
    for (long long j = i; j <= n; j++) {
      long long k = n / i;
      k /= j;
      ans += k - j + 1;
    }
  }
  cout << ans << endl;
  return 0;
}

File: ABC-227/C.py
n = int(input())

if n == 1:
    print(1)
    exit()
ans = 0
for i in range(1, int(n**0.5)):
    for j in range(i, int(n**0.5) + 1):
        k = n // (i * j)
        if k < j:
            break
        ans += k - j + 1

print(ans)
